contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -252,40 +248,49 @@ def get_or_create_language(language):
     )
 
 
-def create_content_thumbnail(thumbnail_string, file_format_id=file_formats.PNG, preset_id=None, uploaded_by=None):
-    thumbnail_data = ast.literal_eval(thumbnail_string)
-    if thumbnail_data.get('base64'):
-        with tempfile.NamedTemporaryFile(suffix=".{}".format(file_format_id), delete=False) as tempf:
-            tempf.close()
-            write_base64_to_file(thumbnail_data['base64'], tempf.name)
-            with open(tempf.name, 'rb') as tf:
-                return create_file_from_contents(tf.read(), ext=file_format_id, preset_id=preset_id, uploaded_by=uploaded_by)
+def create_associated_thumbnail(ccnode, ccfilemodel):
+    encoding = None
+    try:
+        encoding = ccnode.thumbnail_encoding and json.loads(ccnode.thumbnail_encoding).get('base64')
+    except ValueError:
+        logging.error("ERROR: node thumbnail is not in correct format ({}: {})".format(ccnode.id, ccnode.thumbnail_encoding))
+
+    # Save the encoding if it doesn't already have an encoding
+    if not encoding:
+        encoding = get_thumbnail_encoding(str(ccfilemodel))
+        ccnode.thumbnail_encoding = json.dumps({
+            "base64": encoding,
+            "points": [],
+            "zoom": 0,
+        })
+        ccnode.save()
+
+    return create_content_thumbnail(encoding, uploaded_by=ccfilemodel.uploaded_by, file_format_id=ccfilemodel.file_format_id, preset_id=ccfilemodel.preset_id)
 
 
 def create_associated_file_objects(kolibrinode, ccnode):
     logging.debug("Creating LocalFile and File objects for Node {}".format(kolibrinode.id))
     for ccfilemodel in ccnode.files.exclude(Q(preset_id=format_presets.EXERCISE_IMAGE) | Q(preset_id=format_presets.EXERCISE_GRAPHIE)):
         preset = ccfilemodel.preset
-        format = ccfilemodel.file_format
```

Comment:
Don't overwrite built-in format method

Reply:
good catch!  maybe now that we have the strict PEP8 rules about how multiple imports per line are not allowed, we can discuss a convention for importing modules with short names to avoid these kinds of situations!

it'd be nice to free up common words like "format" for local variable names.